www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -85,14 +85,29 @@
                 </Model>
             </accesslist>
             <abort type="BooleanField"/>
-            <DisableSuperuser type="BooleanField"/>
+            <DisableSuperuser type="OptionField">
+                <BlankDesc>root (default)</BlankDesc>
+                <OptionValues>
+                    <www value="1">www</www>
```

Comment:
I assume this is for backwards compat?

Reply:
It might be a better idea to make this required too and map 0 and 1 explicitely. Then it would be 100% the same as the checkbox before.